src/VisualStudio/CSharp/Impl/CodeModel/CSharpCodeModelService.cs

```diff
@@ -1140,7 +1140,8 @@ private EnvDTE.vsCMAccess GetDefaultAccessibility(SyntaxNode node)
                 }
             }
 
-            if (node is AccessorDeclarationSyntax)
+            if (node is AccessorDeclarationSyntax || 
+                node is ArrowExpressionClauseSyntax)
             {
                 return GetAccess(node.FirstAncestorOrSelf<BasePropertyDeclarationSyntax>());
```

Comment:
Sure, but isn't this returning the access for the property, not for the accessor?